lib/core/utils/flattened-tree.js

```diff
@@ -123,6 +123,9 @@ axe.utils.getFlattenedTree = function (node, shadowId) {
 axe.utils.getNodeFromTree = function (vNode, node) {
 	var found;
 
+	if (vNode.actualNode === node) {
```

Comment:
Since I'm still trying to keep these straight in my head, can you elaborate on what problem this solves?